web-server/v0.4/src/services/dashboard.js

```diff
@@ -134,8 +134,8 @@ export async function queryIterations(params) {
 
   let iterationRequests = [];
   selectedResults.map(result => {
-    const controller_dir = result['@metadata.controller_dir'];
-    if (controller_dir === undefined) {
+    let controller_dir = result['@metadata.controller_dir'];
```

Comment:
This was already fixed, so a rebase will get the proper fix already committed.

Reply:
Sounds good! I'll go ahead and rebase in order to resolve this.